hi-ml-azure/src/health_azure/datasets.py

```diff
@@ -123,19 +124,18 @@ def to_input_dataset_local(self, workspace: Optional[Workspace]) -> Tuple[Option
         returned.
 
         :param workspace: The AzureML workspace to read from.
-        :return: Pair of optional path to dataset and optional mountcontext.
+        :return: Pair of path to dataset and optional mountcontext.
```

Comment:
In the type signature the path to dataset seems to still be optional? Also nit: should be 'paths' plural